src/screens/buy/buy.tsx

```diff
@@ -0,0 +1,123 @@
+import { TouchableOpacity } from '@gorhom/bottom-sheet';
+import React from 'react';
+import { Text, View } from 'react-native';
+import { isTablet } from 'react-native-device-info';
+import { useDispatch } from 'react-redux';
+
+import { Divider } from '../../components/divider/divider';
+import { Icon } from '../../components/icon/icon';
+import { IconNameEnum } from '../../components/icon/icon-name.enum';
+import { OctopusWithLove } from '../../components/octopus-with-love/octopus-with-love';
+import { Quote } from '../../components/quote/quote';
+import { RobotIcon } from '../../components/robot-icon/robot-icon';
+import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { TextSegmentControl } from '../../components/segmented-control/text-segment-control/text-segment-control';
+import { WhiteContainer } from '../../components/white-container/white-container';
+import { WhiteContainerAction } from '../../components/white-container/white-container-action/white-container-action';
+import { WhiteContainerDivider } from '../../components/white-container/white-container-divider/white-container-divider';
+import { WhiteContainerText } from '../../components/white-container/white-container-text/white-container-text';
+import { useResetDataHandler } from '../../hooks/use-reset-data-handler.hook';
+import { ThemesEnum } from '../../interfaces/theme.enum';
+import { ScreensEnum } from '../../navigator/enums/screens.enum';
+import { useNavigation } from '../../navigator/hooks/use-navigation.hook';
+import { changeTheme } from '../../store/settings/settings-actions';
+import { useThemeSelector } from '../../store/settings/settings-selectors';
+import { useSelectedAccountSelector } from '../../store/wallet/wallet-selectors';
+import { formatSize } from '../../styles/format-size';
+// import { SettingsHeader } from './settings-header/settings-header';
+import { useBuyStyles } from './buy.styles';
+
+export const Buy = () => {
+  const styles = useBuyStyles();
+  const dispatch = useDispatch();
+  const { navigate } = useNavigation();
+  const handleLogoutButtonPress = useResetDataHandler();
+
+  const theme = useThemeSelector();
+  const publicKeyHash = useSelectedAccountSelector().publicKeyHash;
+
+  const selectedThemeIndex = theme === ThemesEnum.light ? 0 : 1;
+
+  const handleThemeSegmentControlChange = (newThemeIndex: number) =>
+    dispatch(changeTheme(newThemeIndex === 0 ? ThemesEnum.light : ThemesEnum.dark));
+
+  return (
+    <>
+      {/* <SettingsHeader /> */}
```

Comment:
Are we need this here?

Reply:
its slightly outdated commit commented, already fixed